save-backend/src/main/kotlin/com/saveourtool/save/backend/service/vulnerability/VulnerabilityService.kt

```diff
@@ -434,6 +455,36 @@ class VulnerabilityService(
         .orNotFound { "Could not find date for ${dateDto.vulnerabilityName} of type ${dateDto.type} (${dateDto.date})." }
         .let { vulnerabilityDateRepository.delete(it) }
 
+    /**
+     * @param vulnerabilityName [Vulnerability.name]
+     * @param tag tag to add
+     * @return updated [Vulnerability]
+     */
+    @Transactional
+    fun addTag(vulnerabilityName: String, tag: String): Vulnerability {
+        val vulnerability = vulnerabilityRepository.findByName(vulnerabilityName).orNotFound {
+            "Could not find vulnerability $vulnerabilityName"
+        }.apply {
+            tags = getTags().plus(tag).joinToString(DATABASE_DELIMITER)
```

Comment:
suggestion to sort them before saving in database to have similar values for different vulnerabilities when similar tags added in different times\order

Reply:
Agree, sorting is fine

Reply:
Implemented tag sorting